GOVERNANCE.md

```diff
@@ -12,49 +12,98 @@ The WG has final authority over this project including:
 * Conduct guidelines
 * Maintaining the list of additional Collaborators
 
-For the current list of WG members, see the project README.md.
+For the current list of WG members, see [the project README.md][nodejs-docs-].
 
-## Collaborators
+## Membership
 
-The Node.js Documenation WG GitHub repository is maintained by the WG and
-additional Collaborators who are added by the WG on an ongoing basis.
+### Joining the WG
 
-Individuals making significant and valuable contributions are made Collaborators
-and given commit-access to the project. These individuals are identified by the
-WG and their addition as Collaborators is discussed over GitHub issues - at time
-of writing, these individuals are encouraged to comment on [this specific
-issue][].
+Membership may be extended by current WG members to individuals who have made
+contributions to documentation in the Node or JavaScript communities who **have
+expressed interest in membership**. Candidates for membership will be approved
+or denied by existing WG members use the [Consensus Seeking Process][consensus]
+process below. Individuals may express interest publicly on the [sign-up
+issue][sign-up] or privately to any existing Docs WG member. Put simply: **if
+you are a documentarian and are willing to fulfill the [responsibilities of
+membership][membership-responsibilities] to the best of your ability, the Docs
+WG would be happy to receive your help.**
+
+The Docs WG encourages prospective members to read the
+[CONTRIBUTING.md][nodejs-doc-contributing] and
+[GETTING-STARTED.md][nodejs-doc-getting-started] documents for an idea of how
+the WG operates.
+
+### Membership Access and Responsibilities
+
+Membership in the Node.js Documentation WG entails the following access:
+
+* Membership in the [Node.js Documentation Slack][nodejs-doc-slack].
+* Membership in the **@nodejs/documentation** GitHub team.
+* Collaboratorship on the [Node.js Documentation Repository][nodejs-doc-repo].
+
+Members should prepare to take part in the following activities, to the best of
+their abilities:
+
+* Attend meetings in the form of Google Hangouts.
+* Coordinate via [**nodejs/docs** issues][nodejs-doc-repo].
+* Coordinate with WG members via [Slack][nodejs-doc-slack].
+* Respond to pull requests, and issues on the **nodejs/node** tracker that:
```

Comment:
Should WG members that are **not** collaborators on nodejs/node still do this? The [nodejs/node collaborator guide](https://github.com/nodejs/node/blob/master/COLLABORATOR_GUIDE.md#accepting-modifications) isn't very clear on non-collaborator review.